components/style/keyframes.rs

```diff
@@ -70,28 +72,7 @@ impl KeyframeSelector {
 #[cfg_attr(feature = "servo", derive(HeapSizeOf))]
 pub struct Keyframe {
     pub selector: KeyframeSelector,
-    pub declarations: Arc<Vec<PropertyDeclaration>>,
-}
-
-impl Keyframe {
-    pub fn parse(context: &ParserContext, input: &mut Parser) -> Result<Keyframe, ()> {
-        let percentages = try!(input.parse_until_before(Delimiter::CurlyBracketBlock, |input| {
-            input.parse_comma_separated(|input| KeyframePercentage::parse(input))
-        }));
-        let selector = KeyframeSelector(percentages);
-
-        try!(input.expect_curly_bracket_block());
-
-        let declarations = input.parse_nested_block(|input| {
-            Ok(parse_property_declaration_list(context, input))
-        }).unwrap();
-
-        // NB: Important declarations are explicitely ignored in the spec.
-        Ok(Keyframe {
-            selector: selector,
-            declarations: declarations.normal,
-        })
-    }
+    pub declarations: Arc<Vec<(PropertyDeclaration, Importance)>>,
```

Comment:
Important declarations in keyframes are ignored, so the previous implementation was correct. Please make this just a `Vec<PropertyDeclaration>`, and point to https://drafts.csswg.org/css-animations/#keyframes:

> None of the properties interact with the cascade (so using !important on them is invalid and will cause the
> property to be ignored).